api/settings.md

```diff
@@ -180,6 +191,18 @@ These feature flags can be different for each operation systems.
 recommended to be updated. Current version still works.
 * `force_update` _Android_ specific: Is enabled if current application version is not supported anymore.
 
+#### Provider
+
+Info for parking space provider. Key is linked with [location endpoint](/api/locations).
+
+ Node       | Description
+---         | ---
+`image`     | Large provider logo.
+`thumbnail` | Provider logo thumbnail
+`header`    | Translated header text for required fields
+`legal`     | Contains two url lists for terms and conditions. <br>- `accept`: User needs to accept t&c with some action (like checkbox).  <br>- `shown`: User needs only to see the t&c.
```

Comment:
This maybe is a legal issue, how do we make sure that a user accepted the terms and conditions if a third party provider does the booking (e.g. inrix)?